main.go

```diff
@@ -17,48 +17,43 @@ var Version = "unknown"
 func main() {
 	var local, global bool
 	var err error
-	// Load environment
-	err = godotenv.Load(".env")
+	err = godotenv.Load(".env") // Load .env
 	if err != nil {
 		global = false
 	} else {
 		global = true
 	}
-	// Load .env file for production
-	err = godotenv.Overload(".env.local") // Load .env.local for developing
+
+	err = godotenv.Overload(".env.local") // Load .env.local
 	if err != nil {
 		local = false
 	} else {
 		local = true
 	}
```

Comment:
NITPICK: could just be `local = (err == nil)` and same above

Reply:
Fixed